### ISSUE TITLE
`--install` from within a project fails

### ISSUE THREAD

**Issue by sergeio** [Khan/todo-tools#2]
Currently, we expect to only run `todo_tools.py --install` from within the `todo_tools/` directory, but it would be nice if we could do it from within the project we want to install the script in.

Doing this now results in the following output.

```
(khan27)khan/webapp$ todo_tools.py --install .
Traceback (most recent call last):
  File ".../todo-tools/todo_tools.py", line 154, in <module>
    sys.exit(main())
  File ".../todo-tools/todo_tools.py", line 43, in main
    '.git/hooks/post-commit'))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py", line 119, in copy
    copyfile(src, dst)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: './todo_tools.py'
```

@willzfarmer <3

**Comment by TheDataLeek** [comment]
See PR #3

**Comment by sergeio** [comment]
Closed by #3
